grpc-google-cloud-automl-v1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <!-- TODO: remove after 1.1.0 released -->
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/language/v1/*Grpc</className>
```

Comment:
This needs to be automl

Reply:
Oh, copy pasted in two places, but replaced the name of the service only in one. Updated